Mobile/ShopLens/Droid/Notifications/ErrorDialogCreator.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Android.App;
+using Android.Content;
+using System;
+
+namespace ShopLens.Droid.Notifications
+{
+    public class ErrorDialogCreator
+    {
+        private AlertDialog.Builder alertDialog;
+
+        public ErrorDialogCreator (Context context, string title, string message, string neutralButton) : this(context, title, message)
```

Comment:
No reason to create an object just to call its constructor.

Just make this a static method